reference/common-configuration.md

```diff
@@ -71,4 +73,8 @@ CC_WORKER_COMMAND_1=my-other-worker
 ```
 
 By default, workers will be restarted if they exit with an error code. You can customise this behavior by setting the
-environment variable `CC_WORKER_RESTART` to one of `always`, `on-failure` (the default) or `no`.
+environment variable `CC_WORKER_RESTART` to one of `always`, `on-failure` (the default) or `on-failure`.
+
+You can define a delay to restart your worker with the environement variable `CC_WORKER_RESTART_DELAY`, the value is in seconds with a default value of `1`. It will apply to all registered workers.
+
+For example if your worker exits too quickly, even with a restart policy set to `always` Systemd will not restart it after several failed attempts.
```

Comment:
```suggestion
For example, if your worker exits too quickly, even with a restart policy set to `always`, Systemd will not restart it after several failed attempts.
```

Reply:
I don't really like this example as the restart delay is not really intended for that, even if it could work.

Reply:
Do you have another usage example or I just remove this line ?

Reply:
Maybe you can make it a warning regarding the burst limit, something as:

`If the CC_WORKER_RESTART_DELAY value is too low and the restart policy is set to always, your worker might hit the restart burst limit, which may prevent your worker from being correctly restarted. If you want to have a restart delay less than 1 second and expect your script to restart multiple times in a very short time, please let us know on our support. Additional control over the burst properties might be exposed in the future `

Reply:
Thanks !